src/app.rs

```diff
@@ -569,6 +570,38 @@ impl<'a, 'b, T: State + 'a> ApplicationBuilder<'a, 'b, T> {
         self
     }
 
+    /// Add a given ECS bundle to the game loop.
+    ///
+    /// A bundle is a container for registering a bunch of ECS systems and their dependent
+    /// resources and components.
+    ///
+    /// # Parameters
+    ///
+    /// - `bundle`: The bundle to add
+    ///
+    /// # Returns
+    ///
+    /// This function returns ApplicationBuilder after it has modified it, this is
+    /// wrapped in a `Result`.
+    ///
+    /// # Errors
+    ///
+    /// This function creates systems and resources, which use any number of dependent
+    /// crates or APIs, which could result in any number of errors.
+    /// See each individual bundle for a description of the errors it could produce.
+    ///
+    pub fn with_bundle<B>(mut self, bundle: B) -> Result<Self>
+    where
+        B: for<'c> ECSBundle<'a, 'b, (&'c EventsLoop)>,
```

Comment:
Parentheses are unnecessary. If you want to make a tuple with single element you need to make it like this `(A,)`